.github/workflows/test.yaml

```diff
@@ -89,7 +89,7 @@ jobs:
         run: |
           MATRIX=$(jq -nsc '{
               "os": ["ubuntu-latest", "macos-latest", "windows-latest"],
-              "environment": ["test-38", "test-39", "test-310", "test-311", "test-312"],
+              "environment": ["test-39", "test-310", "test-311", "test-312"],
```

Comment:
We should likely update to add `test-313`, though let's dot this in another PR.

Reply:
Yes, the distinction isn't that important for Colorcet, but for other packages, the core is the absolute minimum of dependency (so we can check the latest), whereas test-3XX has a lot of dependencies (which can have pins on other dependencies).